src/components/text-editor/text-editor.scss

```diff
@@ -48,10 +50,21 @@
 }
 
 :host(limel-text-editor:focus-within),
-:host(limel-text-editor:hover) {
+:host(limel-text-editor:not([ui='compact']):hover) {
```

Comment:
should we say 'default' instead of not([ui='compact'])?

Reply:
no. because we want this when the ui is not compact but is hovered. there is something for that when it is compact and hovered further down

Reply:
I meant like this: 
:host(limel-text-editor:([ui='default']):hover)

Reply:
I know what you mean :) . I think this is more correct. If we ever make other `ui` types, we don't want to mention their names every time here, and keep track of this. This way we're just saying for anything that is not the `minimal`, we consider this styles. It doesn't matter what.. but just _not_ the `minimal` ui

Reply:
hm, yes, from this point of you, it makes sense. agree 😊